### ISSUE TITLE
Fire an event when a specific message is received

### ISSUE THREAD

**Issue by gemboot** [OrbitalDan/EventghostSkypePlugin#1]
Hello,

Thank you for this great plugin !! :) very useful !

I wanted to know if you had any plan on enhancing this plugin to make it more versatile, would it be possible for you to generate events when a specific message is received from a specific person ? It would be a great feature I guess ! Also would it be possible to set the online status  (Do not disturb/Online/Away/Invisible/Offline) triggered with a an event ? it would be very useful as well.

Thanks again !!

**Comment by OrbitalDan** [comment]
Okay, I think the COM API can support this operation, but it may take a while before I can implement this.

**Comment by OrbitalDan** [comment]
I've split the status action into a separate issue. ( #2 )

**Comment by gemboot** [comment]
Thank you so much for your kind reply, and for taking those requests into consideration, I really hope you will have some spare time to work on this, I'm pretty sure it will be useful to many people.

You did well by splitting the 2 requests, if you ask me which feature I want first, I would say the one with the Skype Online Status ;) but of course you do what you want.

Thank you again !

**Comment by gemboot** [comment]
Hi there,

I was just wondering if you had the time in the past 6 months to work on those features ? especially the "Skype Online Status" one ? It would be so nice to have this feature ! :)

Thank you.

**Comment by gemboot** [comment]
hi :)
a year has past, any news regarding this request ? if you abandoned the project that's fine, just say it ;) 
Thank you

**Comment by OrbitalDan** [comment]
Yeah, I don't have time to work on this anymore, Microsoft is killing the
APIs with no clear plans to replace them, and I just had to decommission
the HTPC I was using as my primary testing device.  Last I checked,
Microsoft was trying to turn Skype into some sort of chatbot cloud
platform, but had not yet rolled out the APIs and development tools.  I'm
abandoning the project, because it will need a complete re-write to adapt.
Please feel free to fork it if you want to try to develop it yourself.  If
you decline updates to Skype, you may be able to keep the COM API that this
plugin uses for a while, but I just don't see a future for it.
